parser-typechecker/src/Unison/Codebase/Editor/HandleCommand.hs

```diff
@@ -144,24 +141,11 @@ commandLine config awaitInput setBranchRef rt notifyUser notifyNumbered loadSour
     GetTermsMentioningType ty -> Codebase.termsMentioningType codebase ty
     CodebaseHashLength -> Codebase.hashLength codebase
     -- all builtin and derived type references
-    TypeReferencesByShortHash sh -> do
-      fromCodebase <- Codebase.typeReferencesByPrefix codebase sh
-      let fromBuiltins = Set.filter (\r -> sh == Reference.toShortHash r)
-            $ B.intrinsicTypeReferences
-      pure (fromBuiltins <> Set.map Reference.DerivedId fromCodebase)
+    TypeReferencesByShortHash sh -> Backend.typeReferencesByShortHash codebase sh
```

Comment:
Eventually we want most (all?) commands to delegate to `Backend`.

Reply:
Things that are super specific to the CLI probably should not go into `Backend`.